swan-cern/scripts/before-notebook.d/12_config.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+# Additional configuration that is needed for the CERN SWAN instance is made here.
+
+# Add support for VOMS
```

Comment:
You can remove this comment, it's superseded by the introductory comment of the script above.